lib/ioki/apis/operator_api.rb

```diff
@@ -35,6 +35,12 @@ class OperatorApi
         base_path:   [API_BASE_PATH, 'admin'],
         model_class: Ioki::Model::Operator::Permission
       ),
+      Endpoints.crud_endpoints(
+        :admin,
+        base_path:   [API_BASE_PATH],
+        model_class: Ioki::Model::Operator::Admin,
+        except:      [:create, :update, :delete, :index]
```

Comment:
so we basically have
show

Reply:
yes 👍 For now! But maybe at some point later we want to add more endpoints.